Content.Server/Singularity/EntitySystems/GravityWellSystem.cs

```diff
@@ -61,201 +58,226 @@ public override void Update(float frameTime)
         foreach(var (gravWell, xform) in EntityManager.EntityQuery<GravityWellComponent, TransformComponent>())
         {
             var curTime = _timing.CurTime;
-            if (gravWell.NextPulseTime <= curTime)
-                Update(gravWell.Owner, curTime - gravWell.LastPulseTime, gravWell, xform);
+            if (gravWell.NextScanTime <= curTime)
+                Update(gravWell.Owner, curTime - gravWell.LastScanTime, gravWell, xform);
         }
     }
 
     /// <summary>
-    /// Makes a gravity well emit a gravitational pulse and puts it on cooldown.
-    /// The longer since the last gravitational pulse the more force it applies on affected entities.
+    /// Updates the set of captured entities for a gravity well.
     /// </summary>
-    /// <param name="uid">The uid of the gravity well to make pulse.</param>
-    /// <param name="gravWell">The state of the gravity well to make pulse.</param>
-    /// <param name="xform">The transform of the gravity well to make pulse.</param>
+    /// <param name="uid">The uid of the gravity well to make scan.</param>
+    /// <param name="gravWell">The state of the gravity well to make scan.</param>
+    /// <param name="xform">The transform of the gravity well to make scan.</param>
     private void Update(EntityUid uid, GravityWellComponent? gravWell = null, TransformComponent? xform = null)
     {
         if (Resolve(uid, ref gravWell))
-            Update(uid, _timing.CurTime - gravWell.LastPulseTime, gravWell, xform);
+            Update(uid, _timing.CurTime - gravWell.LastScanTime, gravWell, xform);
     }
 
     /// <summary>
-    /// Makes a gravity well emit a gravitational pulse and puts it on cooldown.
+    /// Updates the set of captured entities for a gravity well.
     /// </summary>
-    /// <param name="uid">The uid of the gravity well to make pulse.</param>
-    /// <param name="gravWell">The state of the gravity well to make pulse.</param>
-    /// <param name="frameTime">The amount to consider as having passed since the last gravitational pulse by the gravity well. Pulse force scales with this.</param>
-    /// <param name="xform">The transform of the gravity well to make pulse.</param>
+    /// <param name="uid">The uid of the gravity well to make scan.</param>
+    /// <param name="gravWell">The state of the gravity well to make scan.</param>
+    /// <param name="frameTime">The amount to consider as having passed since the last gravitational scan by the gravity well. Scan force scales with this.</param>
+    /// <param name="xform">The transform of the gravity well to make scan.</param>
     private void Update(EntityUid uid, TimeSpan frameTime, GravityWellComponent? gravWell = null, TransformComponent? xform = null)
```

Comment:
frametime is unused.

Reply:
Switched to current time so we only resolve it once. Might be slightly faster.